modules/page-memory/src/test/java/org/apache/ignite/internal/pagememory/persistence/TestPartitionMeta.java

```diff
@@ -68,15 +69,22 @@ public void writeTo(PartitionMetaIo metaIo, long pageAddr) {
      */
     public static class TestPartitionMetaIo extends PartitionMetaIo {
         /** I/O versions. */
-        public static final IoVersions<TestPartitionMetaIo> VERSIONS = new IoVersions<>(new TestPartitionMetaIo(1));
+        public static final IoVersions<TestPartitionMetaIo> VERSIONS = new IoVersions<>(new TestPartitionMetaIo(7, 1));
```

Comment:
Why 7? If you wan't to use the constant from the real IO, let's do it. If not, a proper explaining constant is needed.

Reply:
done. Never tested code with anything but 7 here, so decided not to change it. But all tests pass with anything >0